spark-jobs/src/main/scala/za/co/absa/enceladus/common/config/JobParser.scala

```diff
@@ -45,10 +45,10 @@ trait JobConfig[R] {
   def keytabFile: Option[String]
 }
 
-object JobConfig {
+object JobParser {
```

Comment:
why remove the config? This does not parse job or standardization. It parses config for the job of a standardization.

```suggestion
object JobConfigParser {
```

or  (but this might be too wordy)

```suggestion
object JobCommandLineParser {
```